SimPPS/PPSPixelDigiProducer/interface/RPixSignalPoint.h

```diff
@@ -0,0 +1,32 @@
+#ifndef SimPPS_PPSPixelDigiProducer_RPix_SignalPoint_H
+#define SimPPS_PPSPixelDigiProducer_RPix_SignalPoint_H
+
+#include "DataFormats/GeometryVector/interface/LocalPoint.h"
+#include "DataFormats/GeometryVector/interface/LocalVector.h"
+
+/**
+ * An elementar charge point, with position, sigma from diffusion and tof.
+ */
+class RPixSignalPoint {
+public:
+  RPixSignalPoint() : pos_(0, 0), sigma_(0), charge_(0){};
+
+  RPixSignalPoint(double x, double y, double s, double charge) : pos_(x, y), sigma_(s), charge_(charge){};
+
+  inline LocalPoint Position() const { return pos_; }
+  inline LocalPoint &Position() { return pos_; }
+  inline double X() const { return pos_.x(); }
+  inline double Y() const { return pos_.y(); }
+  inline double Sigma() const { return sigma_; }
```

Comment:
@mundim , here as well: not needed duplication of methods.

Reply:
It is really needed both kinds of methods, otherwise, a deeper analysis in the code will be needed. Removing one of them (either one) produces compilation errors.

Reply:
Done. Replaced the reference method by a setter method.